Subject: [PATCH] drm/i915: Extend Fake HDMI live status to Valley View

On Atom E38xx based boards from two different manufacturers we have 
encountered the HDMI display being disabled early during the boot process.

Extending the Fake HDMI live status to include Valley View chipsets 
fixes this problem on both boards.

Signed-off-by: James Stafford <jstafford at ampltd.com>
---
  drivers/gpu/drm/i915/intel_hdmi.c | 2 +-
  1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_hdmi.c 
b/drivers/gpu/drm/i915/intel_hdmi.c
index 6b52c6a..358f9b2 100644
--- a/drivers/gpu/drm/i915/intel_hdmi.c
+++ b/drivers/gpu/drm/i915/intel_hdmi.c
@@ -1507,7 +1507,7 @@ intel_hdmi_detect(struct drm_connector *connector, bool force)
 		 * So consider live_status only for certain platforms, for
 		 * others, read EDID to determine presence of sink.
 		 */
-		if (INTEL_INFO(dev_priv)->gen < 7 || IS_IVYBRIDGE(dev_priv))
+		if (INTEL_INFO(dev_priv)->gen < 7 || IS_IVYBRIDGE(dev_priv) || IS_VALLEYVIEW(dev_priv))
 			live_status = true;
 	}

-- 
2.7.4
